frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+import { FilterSet, FilterValue, RangeFilterValue } from "../../models/paged_items";
+
+interface FiltersInfosProps {
+    filters: FilterSet
+}
+
+const FiltersInfos = ({ filters }: FiltersInfosProps) => {
+    const appliedFilters = Object.keys(filters).filter((key: any) => filters[key]?.value)
+    const getValue = (key: any) => {
+        const filterValue: FilterValue = filters[key].value ?? {}
+        const description: any = filters[key].description
+        let valueJSX: any;
+        const descriptionJSX: any = (
+            <>
+                {
+                    <strong>{description.label}: </strong>
+                }
+            </>
+        );
+        const valuesArray = [filterValue]
+        let labels: any = [];
+        let value = ""
+        switch (description.type) {
+            case FILTER_TYPE.SELECT: {
+                labels = valuesArray.map((val: any) => {
+                    if (description.options) {
+                        const option = description.options.find((option: { value: any }) => option.value === val)
```

Comment:
Freezeman crashed (white screen of death) when I selected a Sample Kind filter (`BLOOD`) and then hovered the mouse over the filter info.